packages/desktop/components/popups/CreateDelegationPopup.svelte

```diff
@@ -91,26 +97,34 @@
         }
     }
 
-    async function prepareDelegationOutput(): Promise<void> {
-        try {
-            if (!accountAddress || !rawAmount || !validAmount) {
+    let updatingTransactionInfo = false
+    function prepareDelegationOutput(): void {
+        updatingTransactionInfo = true
+        if (!updatingTransactionInfo) {
+            transactionInfo.preparedTransaction = undefined
+            transactionInfo.preparedTransactionError = undefined
+        }
+        debounce(updateTransactionInfo, 500)()
```

Comment:
This isn't really debouncing anything, it's just a `setTimeout(updateTransactionInfo, 500)`

Reply:
You would need to define the debouncer outside of this function and then call it here. something like:

```ts
const txUpdater = debounce(updateTransactionInfo, 500)

function prepareDelegationOutput(): void {
     txUpdater(transactionInfo)
     ...
} 
     
async function updateTransactionInfo(transactionInfo: ITransactionInfoToCalculateManaCost): Promise<void> {
     ...
}
```